doc/deployment.rst

```diff
@@ -213,15 +213,14 @@ the tables.  It will read in that connection url from
 
 Start the ``fedmsg-hub`` daemon, which will pick up the datanommer plugin,
 which will in turn read in that connection string, start listening for
-messages, and store them all in the db.
-
-::
+messages, and store them all in the db.::
```

Comment:
The changes in this file aren't related to the rest of the commit and don't seem necessary to me. Please don't include them in this commit. Thanks!